### ISSUE TITLE
Crowdin Translations NOT working

### ISSUE THREAD

**Issue by Hetian97** [dvanoni/notero#624]
There is something not working 
Please see the attached screenshot.
Thanks!
![image](https://github.com/user-attachments/assets/1624f7ee-c24a-411d-83f6-148dc64cb978)
![image](https://github.com/user-attachments/assets/aedc75c5-02d9-43d4-a908-e2d127173dab)

**Comment by dvanoni** [comment]
Hi @Hetian97, I see you're looking at this on your fork of Notero. If you look into the details of [one of the jobs](https://github.com/Hetian97/notero/actions/runs/12570866271/job/35041131706), you can see that some required inputs are missing. This is because those inputs are configured in my repo settings and likely aren't configured in your fork.

Can I ask what you're trying to do? If you'd like to submit a PR, I think you can safely ignore those failed jobs because they should run correctly in my repo when the PR is submitted.

If you'd like, I think you can disable the Crowdin job from running every day via the menu:

![CleanShot 2025-01-01 at 13 48 41@2x](https://github.com/user-attachments/assets/5fd8b36c-759c-4352-a4a1-02805b4d541f)

**Comment by Hetian97** [comment]
Thank! I don't know what the Crowdin job is for. Is it okay to disable it?

**Comment by dvanoni** [comment]
Yes, it should be fine to disable the Crowdin job. It's used for updating translations within Notero, and you shouldn't need that on your fork.

**Comment by Hetian97** [comment]
thanks!
